### ISSUE TITLE
meson.build: ERROR: Unknown variable "css_sources_path" with accent colours

### ISSUE THREAD

**Issue by lah7** [ubuntu/yaru#3494]
The community project [`ubuntu-mate-colours`](https://github.com/lah7/ubuntu-mate-colours) previously generated colour variants for Yaru-MATE by patching this repository's `hirsute/mate` branch, but now intends to switch to using this project's build system to generate coloured GTK/Icon themes for Ubuntu MATE 22.04.

From a fresh clone, there seems to be an issue building the accent colours:

```
../gtk/src/meson.build:117:10: ERROR: Unknown variable "css_sources_path".
```
[meson-log.txt.gz](https://github.com/ubuntu/yaru/files/8304786/meson-log.txt.gz)

### Commands
```bash
meson build --prefix=~/Desktop/output
cd build
meson configure -D default=false
meson configure -D dark=false
meson configure -D mate=true
meson configure -D mate-dark=true

meson configure -D buildtype=release
meson configure -D gnome-shell=false
meson configure -D gnome-shell-gresource=false
meson configure -D sounds=false
meson configure -D sessions=false
meson configure -D ubuntu-unity=false

cd ../
ninja -C build install
```

Specifying `meson configure -D accent-colors=` allows the build to succeed, albeit without any accent colours.

Thanks in advance!

**Comment by Muqtxdir** [comment]
hi @lah7 

The `default` and `dark` needs to be set true for build to work now as accent colors are dependent on it.

**Comment by lah7** [comment]
Thanks, it turns out that `gnome-shell=false` causes the error.

Just to clarify, is it possible for accent colours to be generated based on a "flavour"? For example, `Yaru-mate` → `Yaru-mate-blue`?

**Comment by 3v1n0** [comment]
This had been fixed via commit a1548d545304fb27feb04ebefe33d769811b5015, you can now disable gnome-shell again.
